src/pr-info.ts

```diff
@@ -408,7 +408,7 @@ configSuspicious["tsconfig.json"] = (contents, oldText) => {
     // changes only the files array, and all relative paths
     try {
         return !jsonDiff.compare(JSON.parse(oldText || ""), JSON.parse(contents)).every(op =>
-            op.path.startsWith("/files/") && (!("value" in op) || isRelativePath(op.value)))
+            op.path.startsWith("/files/") && (!("value" in op) || isRelativePath(op.value)) || op.path.startsWith("/compilerOptions/paths/"))
```

Comment:
More readable:

```suggestion
            (op.path.startsWith("/files/") && (!("value" in op) || isRelativePath(op.value)))
            || op.path.startsWith("/compilerOptions/paths/"))
```

(Extra parens might not be needed, I just dislike relying on the `&&` precedence...)